### ISSUE TITLE
[FEATURE REQUEST] Identifier for <Message>

### ISSUE THREAD

**Issue by Cicorione** [speedata/publisher#545]
Hi @pgundlach 

I wonder if it might be possible add some sort of identification to the messages output, I am trying to debug some layout issue but I am not so skilled to understand which message is which when I got for each a `nil` value.

Perhaps something:
`<Message id="var1" select="$var1" />` 

Thanks... 🙏

**Comment by pgundlach** [comment]
This is a good feature request, thank you.

A workaround could be to use the lxpath XPath parser which generates output such as 

```
...
Call "Message" from layout (line 22)
Message: "Hello nice world"
Call "PlaceObject" from layout (line 24)
...
```

or identify yourself within message:

    <Message select="concat('color: ', $color)" />

**Comment by Cicorione** [comment]
Thank @pgundlach I didn't realize I could use such workaround... 🙃

**Comment by Cicorione** [comment]
I'd like to point out if the variable is malformed `select="concat('text', $variable)"` shows an empty message without plain text.

**Comment by pgundlach** [comment]
Implemented in version 4.15.17
